crossy-web/app/(app)/play/games/[slug]/gameboard.tsx

```diff
@@ -420,15 +434,33 @@ const Gameboard: React.FC<Props> = ({
                 </text>
               )}
               {/* Friend-is-here indicator */}
-              {friendsCellNumbers.includes(i) && (
-                <circle
-                  cx={col * cellSize + cellSize - 6}
-                  cy={row * cellSize + 6}
-                  r={2.5}
+              {friendsAtThisCell.length === 1 ? (
```

Comment:
Let's 
1. Extract this into a variable `friendsIndicator` at 412
2. Use if/else with null as default instead of ternary